client/src/pages/registration.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+import Form from "../components/Form";
+import Input from "../components/Input";
+import Button from "../components/Button";
+import Heading from "../components/Heading";
+
+const ContentWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  width: 95vw;
+  height: 100vh;
+  background: ${props => props.theme.colors.secondary};
+`;
+
+const Container = styled.div``;
+
+function AttendeeRegistration() {
+  const [state, setState] = React.useState({
+    name: "",
+    address: "",
+    location: "",
+    email: "",
+    phone: "",
+    workshop1: "",
+    workshop2: "",
+    workshop3: "",
+    else: ""
+  });
+
+  function onChange(event) {
+    const value = event.target.value;
+    setState({
+      ...state,
+      [event.target.name]: value
+    });
+  }
+
+  async function handleSubmit(event) {
+    event.preventDefault();
+    await fetch("http://localhost:8080/attendees", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify({
+        state
```

Comment:
to fix the problem I stated before, try something like this:
...
body: JSON.stringify({
        name: state.name, [although Sascha is right about renaming this ;)]
        address: state.address,
...
      })
and everything will be neat and tidy as an object directly in your array ;)

Reply:
Thanks :)